index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Tech Academy</title>
+  <link rel="stylesheet" href="https://use.typekit.net/kjf8squ.css">
+  <script src="https://kit.fontawesome.com/67c66657c7.js"></script>
+  
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header class="primary-header">
+    <div class="container">
+          <nav class="navigation">
+            <div class="menu-button"><img id="humberger" src="img/menu.svg" alt="menu-button"></div>
+            <a href="index.html"><img class="logo"src="img/logo-2.png" alt="Manage" /></a>
+          <div class="nav-bar">
+          <ul  class="nav-list text-neutral-900 fw-bold fs-500" >
+          <li><a class="nav-link"href="index.html">Home</a></li>
+          <li><a class="nav-link" href="about-page.html">About</a></li>
+          <li><a class="nav-link" href="#courses">Courses</a></li>
+          <li><a class="nav-link" href="#instructor">Instrictors</a></li>
+        </ul>
+          </div>
+        
+      </nav>
+    </div>
+  </header>
```

Comment:
### Your Header
![navbar2](https://user-images.githubusercontent.com/93039359/196919874-2f6564b6-a7ba-4a86-a912-b5631b8318ef.png)

### Design Header
![navbars](https://user-images.githubusercontent.com/93039359/196918571-de8dd1a1-4a82-42f8-8756-280e4d82d508.png)

- [ ] You can compare both designs yours and template. You can see in the second image that there is a navbar shown by blue color box which is missing in your design. Kindly add it in your design to follow the design :+1:
- [ ] You can see that there are green lines around your links which are showing margins top and bottom. In the design the margin top and bottom is equal but in your design your links are very close to the bottom but not in center. Kindly make them in margin to follow the design to make your design professional :+1: